src/elements/content-sidebar/__tests__/MetadataSidebarRedesign.test.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { userEvent } from '@testing-library/user-event';
```

Comment:
how do we plan to keep track of packages that have been installed during this development since we dont have the package.json in the branch

Reply:
We’re not adding any new packages except @box/metadata-editor for now. If we need another one, we can figure it out when Dawid’s back from vacation next week.